app/helpers/view_helper.rb

```diff
@@ -64,13 +69,20 @@ def govuk_button_link_to(body = nil, url = nil, html_options = nil, &block)
   end
 
   def govuk_button_to(name, options = {}, html_options = {})
```

Comment:
can we include `&block` as so:
```
def govuk_button_to(name, options = {}, html_options = {}, &block)
```
to help indicate that this method accepts a block?

Reply:
I originally added this, but Rubocop complained:

```
app/helpers/view_helper.rb:71:63: W: Lint/UnusedMethodArgument: Unused method argument - block. If it's necessary, use _ or _block as an argument name to indicate that it won't be used. (https://rubystyle.guide#underscore-unused-vars)
  def govuk_button_to(name, options = {}, html_options = {}, &block)
                                                              ^^^^^
```

Is there a way to make the method explicit, but also appease our linting overlords?

Reply:
Oh, maybe I should just follow it’s advice and prefix this value with `_`…